### DISCUSSION TITLE
discord / cloudflare detection

### DISCUSSION THREAD

**Discussion by Skorchekd** [ZFC-Digital/puppeteer-real-browser#209] [General]
can someone try registering an account on discord.com in the browser? the account is instantly terminated. The reason i wish to have this addressed is because usually discords anti bot is cloudflares anti bot as its all done by cloudflares anti bot systems. I did make an issue but maybe this is better and more relevant as a discussion.

**Comment by RubberArchind** [comment]
Can share some code, i can help you test it

**Reply by Skorchekd** [reply]
simply go to discord.com using the demo code and click continue in browser once page load. input username checkbox select that press continue do hcaptcha and then fill in dob. when it locks and you fill in the info it flags you and terminates account.

**Reply by RubberArchind** [reply]
ok i will try this

**Comment by RubberArchind** [comment]
ok after try it fortunately my account not terminated, but i am unable to login, when i open the network tab i see that the login response was 429 Too many request, idk why this happened but might be a problem with my network, i will try it again with another network soon

**Reply by prettymaya** [reply]
if u try with proxy maybe proxy is bad

**Reply by Skorchekd** [reply]
yeah its detected

**Reply by RubberArchind** [reply]
sorry for late update guys, im not using proxy, and idk why i got that error

**Reply by RubberArchind** [reply]
did you guys solve the hcaptcha manually or using automated services?
